inc/c_util/flags_to_string.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef FLAGS_TO_STRING_H
+#define FLAGS_TO_STRING_H
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/xlogging.h"
+
+#include "c_pal/string_utils.h"
+
+/*introduce a name for function that can be called. The name is based on "X", User code might look like FLAGS_TO_STRING(ISC_REQ)(value)*/
+#define FLAGS_TO_STRING(X) MU_C2(FLAGS_TO_STRING_,X)
+
+/*this macro declares a function to be used in a header*/
+#define FLAGS_TO_STRING_DECLARE_FUNCTION(X) MOCKABLE_FUNCTION(, char*, FLAGS_TO_STRING(X), uint32_t, argument)
```

Comment:
missing stdlib.h

Reply:
Nit sure which f the teokens here is missing from stdlib.h, since uint_32t comes from stdint.h included from intteypes.h